src/command.h

```diff
@@ -183,6 +187,52 @@ class DrawRectCommand : public PipelineCommand {
   float height_ = 0.0;
 };
 
+/// Command to draw a grid of recrangles on screen.
+class DrawGridCommand : public PipelineCommand {
+ public:
+  explicit DrawGridCommand(Pipeline* pipeline, PipelineData data);
+  ~DrawGridCommand() override;
+
+  const PipelineData* GetPipelineData() const { return &data_; }
+
+  void EnableOrtho() { is_ortho_ = true; }
+  bool IsOrtho() const { return is_ortho_; }
+
+  void EnablePatch() { is_patch_ = true; }
+  bool IsPatch() const { return is_patch_; }
+
+  void SetX(float x) { x_ = x; }
+  float GetX() const { return x_; }
+
+  void SetY(float y) { y_ = y; }
+  float GetY() const { return y_; }
+
+  void SetWidth(float w) { width_ = w; }
+  float GetWidth() const { return width_; }
+
+  void SetHeight(float h) { height_ = h; }
+  float GetHeight() const { return height_; }
+
+  void SetColumns(float c) { columns_ = c; }
```

Comment:
Columns and Rows are used as uint32_t, should these just be uint32_t instead of float?

Reply:
Done